### ISSUE TITLE
New Dark Theme colors are heavy on eyes

### ISSUE THREAD

**Issue by kepi** [vector-im/riot-web#8664]
I just received update of riot-web app and I like the redesign. Unfortunately color combination in dark theme is extremely heavy on eyes.

One of main points of dark themes is to safe our eyes, but especially main chat area is complete opposite. In fact it is almost easier on my eyes to switch to light theme.

I'm not sure of exact color combination, but chat area in previous version was ok, list of rooms is ok, but chat area currently is not. Maybe text color is too white, maybe background is too dark...

**Comment by kepi** [comment]
One observation after 30 minutes usage: maybe biggest problem is font size, which is simply too small with used fonts and color combination. I would probably survive it longer without notice when font is bigger.

Is there any option to change size of fonts? I tried to search for it but without success.

**Comment by tompinzler** [comment]
> maybe biggest problem is font size, which is simply too small
I second that

**Comment by fbruetting** [comment]
Yeah, we really need a bigger font size. Older people otherwise will have problems, and still for me with good sight it’s too small for being comfortable. Here’s a comparison with Wire and WhatsApp:

![bildschirmfoto von 2019-02-16 18-31-55](https://user-images.githubusercontent.com/10520767/52903133-c1e39700-3219-11e9-827b-a6a3da9ed8f0.png)

**Comment by progserega** [comment]
Colors was not such contrast in old dark theme. 
New dark them is very heavy for eyes :-( 

![1](https://user-images.githubusercontent.com/1297163/52913015-20d20a80-3305-11e9-953d-e417eea4fc62.png)

**Comment by lampholder** [comment]
We've made dark theme a little less dark - could you take another look and let us know what you think?

**Comment by progserega** [comment]
> We've made dark theme a little less dark - could you take another look and let us know what you think?

Where are I can test it?
On riot.im - steel old dark theme...

**Comment by jryans** [comment]
Do you mean riot.im/develop or riot.im/app? (/develop is newer and likely includes additional tweaks.)

**Comment by progserega** [comment]
> We've made dark theme a little less dark - could you take another look and let us know what you think?
> Do you mean riot.im/develop or riot.im/app? (/develop is newer and likely includes additional tweaks.)

I do not see any color changes in develop version.

**Comment by skerit** [comment]
Huh. I started using Riot a few days ago, so I never saw the old dark theme in action.

I was actually thinking about creating an issue regarding the dark theme being too dark, so it's quite a surprise to see that the previous colors actually looked better :/

**Comment by lampholder** [comment]
We're planning to address this by dividing dark theme into "dark theme" and "black theme".

**Comment by unclechu** [comment]
I'd like to have a feature to set custom CSS in _advanced_ settings section so I could customize specific theme (by selectors like `html.riot-dark-theme` for instance or `html[data-theme=dark]`).

**Comment by aaronraimist** [comment]
@unclechu make sure you put a thumbs up on #4482, that is the issue for that

**Comment by SilverWolf32** [comment]
Personally, I think the current background colors are absolutely perfect! Now, the controls could use some slight tweaking (especially the very bright borders of text boxes and such), but it'd be great if you could leave an option for these background colors as well. (:

**Comment by fti7** [comment]
Its really too dark :-( especially the Box with the Messages.
Checkout the Discord Theming where the Message Box is lighter than the Navigation. Its much better for the Eyes.

![image](https://user-images.githubusercontent.com/7014664/63614363-245f8a80-c5e3-11e9-8b32-90cd3e6e0506.png)


As i know there are Users which really like totally Dark Themes, i would also vote for an Dark and Black Theme split

**Comment by gsbhasin123** [comment]
I really need this, I'm wling to use custom css if anyone knows how (ON the desktop app)
the old one looks awesome, any way to use it rn?

**Comment by abhishekgupta92** [comment]
Any progress on the black theme? :-)

**Comment by 0lm** [comment]
could we also get the ability to edit the colours ourself? the dark theme, as many said, really is heavy on eyes. the current theme is not really dark theme, but it is black theme. black and white contrast is too strong for eyes.
dark would be maybe dark grey? and sepia coloured font? that would be more eye-friendly. maybe add customizable font colors and background colors so we can pick them ourselves?

**Comment by ShadowJonathan** [comment]
From https://github.com/vector-im/element-web/issues/18116; I think the contrast between the background and foreground is a large factor here. Compare the following two;

<details>
<summary>Image</summary>

![image](https://user-images.githubusercontent.com/22740616/126363740-0d476c8d-8786-468a-bda2-70460246e663.png)

</details>

With this;

<details>
<summary>Image</summary>

![image](https://user-images.githubusercontent.com/22740616/126363975-82d97d63-f858-4980-842e-7dcfb867faaa.png)

</details>

Element currently is nearly pure black (`#000000`) on pure white (`#ffffff`), decreasing that contrast a lot would probably help.

Additionally/alternatively; The current theme could be rebranded as a "Pure Black" theme, while a less harsher "grey" theme could be introduced. When GitHub introduced native dark theme switching, it differentiated between its old(?) pure black theme, and a newer greyer theme.

**Comment by Erquint** [comment]
Years later, it is still horritic.  
Just give us custom CSS…
